### ISSUE TITLE
Unable to get detections from outside own receiver network

### ISSUE THREAD

**Issue by PietrH** [inbo/etn#277]
Summary of a reported issue from an email thread on the 2nd of March 2023: @jreubens , @cfmuniz, and others.

- The position of active receivers should always be public.
- The detection info (including position) of animals is visible for everyone with rights to the animal-project (because they tagged the animal) , and for everyone with rights to the receiver-project (because they maintain the receiver).

Marine Gonse reports:

> However I still don't understand how the right access function. I work on the Fish Intel project and one of our tagged seabass was detected by one of Fishowf receiver in Saint-Nazaire (someone from the fishowf project told me that). But when I request the detections of this fish using get_acoustic_detections(animal_project_code = "FISHINTEL", con=con), I can't see the detections of this fish when I filter the acoustic_tag_id == "OPI-602". Does the Fishowf project has particular right access? 

Francisco Hernandez: 

> I can’t find any detections of OPI-602 in any project.

However, there should be 6 detections for that tag :
``` r
etn::get_acoustic_detections(etn::connect_to_etn(), acoustic_tag_id = 'OPI-602')
```

According to @jreubens this might be because currently users can only detect detections from their own receiver networks, if this is the case this would be a bug as the intention is that users can retreive detections for their own projects from any receiver network. 

Another explination could be that Fransisco and Marine possibly do not have sufficient rights to `OPI-602`

These user limits are regulated on the database side. Do I understand this correctly @salvafern ?

**Comment by salvafern** [comment]
No idea honestly - I'm not the admin of the ETN database @bartv and Francisco Hernandez should know more

**Comment by PietrH** [comment]
Whoops, sorry, thanks for the reference!

**Comment by jreubens** [comment]
Hi, Marine should have sufficient rights. She has access to FISHINTEL and as teh tag belongs to that project she should see all detections of that tag...

**Comment by MarineRandon** [comment]
Hi everyone, I am not able to visualize this fish from neither the RStudio server nor the Lifewatch data explorer. There is definitely an issue here. Thank you for your help.

**Comment by CLAUMEMO** [comment]
Hi, 
I am adding to this issue:
Issue raise by user Luis Bentes (lmbentes@gmail.com)
When he checks his detections only those from his own project are shown, but there are another 4 network projects were his tags were detected.

```
> library(etn)
> conLuis <- connect_to_etn(luis credentials) # connecting with Luis' credential (i know... not great)
> conClau <- connect_to_etn(my credentials) #connecting with my credentials, as ETN data manager
> # getting acoustic detections for his project
> TelMarsw_detections_luis <- get_acoustic_detections(conLuis, animal_project_code = "TelMarsw")
> TelMarsw_detections_clau <- get_acoustic_detections(conClau, animal_project_code = "TelMarsw")
> # comparing numer of detections
> nrow(TelMarsw_detections_luis) - nrow(TelMarsw_detections_clau)
[1] -2080
> # checking distinct network projects that appear for Luis 
> unique(TelMarsw_detections_luis$acoustic_project_code)
[1] "TelMarsw"
> # vs for admin
> unique(TelMarsw_detections_clau$acoustic_project_code)
[1] "TelMarsw"      "MOBEIA"        "BECORV"        "Inforbiomares" "PTN-MARSW" 
```

I had raised this issue with Aubri on 2023-01-20, because I was not sure if it was a package issue or a database issue. His answer:

> Issue in the R package, i've send a message to peter
> https://github.com/inbo/etn/commit/7958d31ace58bc317f82c1ea9c0d76876df633a7

Can this be checked soon?

**Comment by PietrH** [comment]
Thanks for digging that up Claudia, I'll look into it this week and see if I can estimate when it'll be fixed. It's on my radar!

**Comment by CLAUMEMO** [comment]
I also don't see how the tag type can affect this.
Testing, I do get different number of detentions depending on the levels of permissions i assign myself. Can we arrange a meeting for this?

**Comment by PietrH** [comment]
@CLAUMEMO any idea if this ever got resolved?

**Comment by CLAUMEMO** [comment]
Yes! Permission issues were refined with the new detection views
